### PR TITLE
TPWDCEE-2285

### PR DESCRIPTION
**Pull request by tatsta**

### CODE REVIEW

File: tests/_support/Helper/Acceptance.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace Helper;
+
+// here you can define custom actions
+// all public methods declared in helper class will be available in $I
+
+class Acceptance extends \Codeception\Module
+{
+    public static function formAuthLink($link, $username, $password)
+    {
```
Comment (jakubpolomsky) [inline tests/_support/Helper/Acceptance.php]: consider using http://php.net/manual/en/function.parse-url.php to correctly pinpoint the correct place for credentials

File: tests/_support/AcceptanceTester.php [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    {
+        $this->wait($seconds);
+    }
+
+    /**
+     * @Then I see in table key :tableKey value :tableValue
+     */
+    public function iSeeInTableKeyValue($tableKey, $tableValue)
+    {
+        //TODO: add searching for "Transaction state" in table and checking the value near it, instead of relying on location
```
Comment (jakubpolomsky) [inline tests/_support/AcceptanceTester.php]: My suggestion would be to do it with javascript. Find the tr containing $tableKey, switch to parent node and search for $tableValue within the node. If found add an unique id to it and search with selenium for it.

File: tests/_support/AcceptanceTester.php [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+    /**
+     * @Given I am on :page page
+     */
+    public function iAmOnPage($page)
+    {
+        // Open the page and initialize required pageObject
+        $this->currentPage = $this->pages[$page];
+
+        $curPage = $this->currentPage;
+        $this->amOnPage($curPage::$URL);
```
Comment (jakubpolomsky) [inline tests/_support/AcceptanceTester.php]: Can we use the `$I` element instead of `$this` ?

File: tests/_support/Helper/Acceptance.php (hunk 1 above)
Comment (tatsta) [reply tests/_support/Helper/Acceptance.php]: Changed that

File: tests/acceptance.suite.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+actor: AcceptanceTester
+modules:
+  enabled:
+    - WebDriver:
+        host: 'hub-cloud.browserstack.com'
+        port: 80
+        url: '%NGROK_URL%/examples'
+        browser: chrome # 'chrome' or 'firefox'
+        capabilities:
+          'build': '3.4.2'
```
Comment (jakubpolomsky) [inline tests/acceptance.suite.yml]: can we have this set to something like 'latest' ?

**Pull request merged**